index.bs

```diff
@@ -129,25 +134,65 @@ The {{Accelerometer/z!!attribute}} attribute of the {{Accelerometer}}
 interface represents the <a>acceleration</a> along Z-axis.
 This attribute returns [=map/value=] for [=latest reading=]["z"] [=map/entry=].
 
-### Accelerometer.includesGravity ### {#accelerometer-includesGravity}
+The LinearAccelerometer Interface {#linear-accelerometer-interface}
```

Comment:
the docs there call it that just in the title :-) the text which follows calls it linear acceleration sensor which is also the name in the API. Maybe they changed the names but forgot to update this one.

I like that it is called LinearAccelerationSensor especially because it is a combined sensor, and LinearAccelerometer makes it sounds like a specialized version of an accelerometer which it is not.